### DISCUSSION TITLE
What are the green-colored circles near the workspaces numbers?

### DISCUSSION THREAD

**Discussion by oren** [regolith-linux/regolith-desktop#1049] [Q&A]
I see green circle near workspace 2 and another one (darker green) near workspace 3. What are they?
![Screenshot from 2024-08-29 20-20-00](https://github.com/user-attachments/assets/f1c48f02-7b56-4257-8a9a-17eeb9b046b5)

**Comment by SoumyaRanjanPatnaik** [comment]
Just some color coding for the workspace. They are included as part of the workspace name.

https://regolith-desktop.com/docs/howtos/change-workspace-icons/
^ Should help you change the label / icons if you want to

**Comment by oren** [comment]
Thank you so much! :heart:
